### PR TITLE
added sidebar and created components for other parts of the page

### PR DESCRIPTION
**Pull request by sonia-ko**

### CODE REVIEW

File: src/pages/AllProductsPage/AllProductsPage.module.css [hunk 1]
```diff
@@ -1,17 +1,19 @@
 .ProductsPageContainer {
```
Comment (Yuliia143) [inline src/pages/AllProductsPage/AllProductsPage.module.css]: Fix className name

File: src/components/sidebar/SideBar.tsx [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
+  };
+
   return (
     <div className={classes.sidebar}>
-      <div>Sidebar</div>
+      <CategoriesFilter />
+      <BrandsFilter />
+      <RatingFilter />
+      <PriceRange defaultMin={0} defaultMax={1000} />
+      <ButtonLight btnText={"Reset"} onClick={resetFilters}></ButtonLight>
```
Comment (Yuliia143) [inline src/components/sidebar/SideBar.tsx]: Fix close tag

File: src/components/sidebar/SideBar.tsx [hunk 3]
```diff
@@ -1,9 +1,22 @@
...
+    console.log("filters reset :) ");
+  };
+
   return (
     <div className={classes.sidebar}>
-      <div>Sidebar</div>
+      <CategoriesFilter />
+      <BrandsFilter />
+      <RatingFilter />
+      <PriceRange defaultMin={0} defaultMax={1000} />
```
Comment (Yuliia143) [inline src/components/sidebar/SideBar.tsx]: Follow one name's type

File: src/pages/AllProductsPage/components/PaginationBlock/PaginationContainer.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+const PaginationContainer: React.FC = () => {
+  const showMoreProducts = () => {
+    console.log("More products shown");
+  };
+
+  return (
+    <div className={classes.container}>
+      <Pagination />
+      <GreenButton btnText={"Show more products"} onClick={showMoreProducts} />
```
Comment (Yuliia143) [inline src/pages/AllProductsPage/components/PaginationBlock/PaginationContainer.tsx]: Create one component for Buttons

File: src/components/footer/socialLinksBlock/LinksList.tsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import classes from "./LinksList.module.css";
+import { Link } from "react-router-dom";
+
+const LinksList: React.FC<{
```
Comment (Yuliia143) [inline src/components/footer/socialLinksBlock/LinksList.tsx]: Create interface
Destructuring for props

**Review by Yuliia143 — CHANGES_REQUESTED** [review]

File: src/pages/AllProductsPage/AllProductsPage.module.css (hunk 1 above)
Comment (sonia-ko) [reply src/pages/AllProductsPage/AllProductsPage.module.css]: Fixed, thanks :)

File: src/components/sidebar/SideBar.tsx (hunk 2 above)
Comment (sonia-ko) [reply src/components/sidebar/SideBar.tsx]: Fixed too, thanks)

File: src/components/sidebar/SideBar.tsx (hunk 3 above)
Comment (sonia-ko) [reply src/components/sidebar/SideBar.tsx]: Renamed, thank you

File: src/pages/AllProductsPage/components/PaginationBlock/PaginationContainer.tsx (hunk 4 above)
Comment (sonia-ko) [reply src/pages/AllProductsPage/components/PaginationBlock/PaginationContainer.tsx]: Done, thank you :)

File: src/components/footer/socialLinksBlock/LinksList.tsx (hunk 5 above)
Comment (sonia-ko) [reply src/components/footer/socialLinksBlock/LinksList.tsx]: Done as well, thanks

**Review by Yuliia143 — APPROVED** [review]

**Comment by sonia-ko** [comment]
Created components of the page.

**Pull request merged**